code_schemes/kakuma_s01e03_reasons.json

```diff
@@ -3,6 +3,102 @@
   "Name": "KAKUMA S01E03 Reasons",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-68025c64",
+      "CodeType": "Normal",
+      "DisplayText": "culture or gender roles",
+      "NumericValue": 1,
+      "StringValue": "culture_or_gender_roles",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-aa4e6411",
+      "CodeType": "Normal",
+      "DisplayText": "training for girls to be wives",
+      "NumericValue": 2,
+      "StringValue": "training_for_girls_to_be_wives",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-0eec27d0",
+      "CodeType": "Normal",
+      "DisplayText": "religion",
+      "NumericValue": 3,
+      "StringValue": "religion",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-53d0dbba",
+      "CodeType": "Normal",
+      "DisplayText": "equal rights",
+      "NumericValue": 4,
+      "StringValue": "equal_rights",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-636d10a6",
+      "CodeType": "Normal",
+      "DisplayText": "gender stereotyping",
+      "NumericValue": 5,
+      "StringValue": "gender_stereotyping",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-30b953f8",
+      "CodeType": "Normal",
+      "DisplayText": "edu - equal opportunities for both",
+      "NumericValue": 6,
+      "StringValue": "edu_equal_opportunities_for_both",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-fdeaf663",
+      "CodeType": "Normal",
+      "DisplayText": "edu - proposed solutions",
+      "NumericValue": 7,
+      "StringValue": "edu_proposed_solutions",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-773d8fdb",
+      "CodeType": "Normal",
+      "DisplayText": "parent level of education or exposure",
```

Comment:
(This is parents in the doc - I think meaning parents').

Reply:
thanks for catching this, updated!